backend/src/recommitment/recommitment.service.ts

```diff
@@ -285,7 +287,9 @@ export class RecommitmentService {
       ...new Set(memberPending.map((itm) => itm.personnel.id)),
     ];
     const membersCommitted = [...memberCommittedBCWS, ...memberCommittedEMCR];
-
+    const membersCommittedBoth = Array.from(new Set([...memberCommittedBCWS.filter((itm) => memberCommittedEMCR.find(r => r.personnel.id === itm.personnel.id)), ...memberCommittedEMCR.filter((itm) => memberCommittedBCWS.find(r => r.personnel.id === itm.personnel.id))]));
```

Comment:
This is kinda a rough line haha. Can you define the two filtered arrays beforehand to clean this up?

Reply:
haha, yes this is yucky.